packages/product-react/stories/Cell.story.tsx

```diff
@@ -0,0 +1,69 @@
+import { storiesOf } from '@storybook/react';
+import * as React from 'react';
+import { Grid } from '../src/components/Grid';
+import '../src/components/Container/Container.scss';
+
+storiesOf('Core Comps | Cell', module)
```

Comment:
This is a nit, but i think we can remove mentions of "core components" when we move stuff into Ray.